Sources/TuistSupport/Utils/Printer.swift

```diff
@@ -1,188 +1,188 @@
-import Basic
-import Foundation
-
-public struct PrintableString: Encodable, Decodable, Equatable {
```

Comment:
RIP: Printer 😢 
Feel free to remove it

Reply:
Going off what your suggestion was this may stick around as it will probably help with colourisation! Not quite ready to go yet 😄